### ISSUE TITLE
UWP UserControls

### ISSUE THREAD

**Issue by wind-rider** [Tapanila/SharpCaster#114]
Hi,

I saw there are several UWP/XAML UserControls in the SharpCaster.Simple application; it could be nice to polish them and put them into a SharpCaster.Controls nuget package.

What do you think?

(I've been working on a Flyout DevicePicker control for selecting a chromecast)

**Comment by Tapanila** [comment]
I did include the controls in SDK project itself at some point. But because of the .net core and support of .net framework I removed them.

Would love to have a new NuGet package that would provide controllers for this stuff
**Playback**
- Video/Audi playback with scrub bar
- Options to display time played, time left and duration.
- Next/Previous buttons
- Skip Forward 15 seconds and Skip Backwards 15 seconds

**Chromecast discovery**

- List of chromecasts
- Ability to refresh and show status (searching, done searching)
- Ability to disconnect from chromecast
